util/Migrator/DbMigrator.cs

```diff
@@ -124,6 +126,27 @@ private bool MigrateDatabase(bool enableLogging = true,
             .WithTransaction()
             .WithExecutionTimeout(new TimeSpan(0, 5, 0));
 
+        if (_hostingEnvironment != null && _hostingEnvironment.IsProduction())
+        {
+            builder = builder.WithTransactionPerScript();
+        }
+        else
+        {
+            builder = builder.WithTransaction();
+        }
```

Comment:
🎨 This is an "if self-host or not" thing for us. We want self-host working as-is with one transaction.

Reply:
added flag with default